### PR TITLE
CVE-2016-2048

### PR DESCRIPTION
**Pull request by cgs8601**
Filled out CVE-2016-2048 - Curtis

### CODE REVIEW

File: cves/CVE-2016-2048.yml [hunk 1]
```diff
@@ -224,11 +257,15 @@ lessons:
...
+    applies: true
     note:
+      An (arguably extraneous) permissions check for the add permission was removed,
+      causing the issue when the other add check was not performed due to the call
+      possesing an object_id.
   least_privilege:
-    applies:
+    applies: true
     note:
+      The "Save as new" operation was being accidentally afforded the add privalage.
```
Comment (gollb310) [inline cves/CVE-2016-2048.yml]: "Privileged" is misspelled.

File: cves/CVE-2016-2048.yml [hunk 2]
```diff
@@ -173,20 +198,25 @@ subsystem:
...
     Are there any interesting commits between your VCC(s) and fix(es)?
 
     Write a brief (under 100 words) description of why you think this commit was
     interesting in light of the lessons learned from this vulnerability. Any
     emerging themes?
+  answer:
+    I did not find any commits relating to this vulnerability, 
+    though one might consider my first VCC something of an interesting commit,
+    as it set the stage for the poor maintainability of that add permissions-check,
+    which undoubtedly asisted the second VCC I listed.
```
Comment (gollb310) [inline cves/CVE-2016-2048.yml:216]: If you found no interesting commits, just put in a brief summary of some of the commits in the meantime to show you went through them.

**Review by gollb310 — COMMENTED** [review]
Good report, insightful and detailed. +3 upvotes.

File: cves/CVE-2016-2048.yml (hunk 2 above)
Comment (cgs8601) [reply cves/CVE-2016-2048.yml:216]: Fixed.

File: cves/CVE-2016-2048.yml (hunk 1 above)
Comment (cgs8601) [reply cves/CVE-2016-2048.yml]: Fixed.

**Pull request merged**